app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+kind: snippet
+name: kickstart_rhsm
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Generate Anaconda RHSM configuration for Red Hat OS registration
+test_on:
+- rhel9_dhcp
+-%>
+<%
+  subman_keys = host_param('kt_activation_keys') || host_param('activation_key')
+  subman_org = host_param('subscription_manager_org')
+  subman_registration = host_param_true?('subscription_manager') || subman_keys
+  subman_hostname = " --server-hostname #{host_param('subscription_manager_server_hostname')}" if host_param('subscription_manager_server_hostname')
```

Comment:
This will allow user to set `hostname` and `rhsm_baseurl` only for RHEL 9,
it's not supported in `app/views/unattended/provisioning_templates/snippet/redhat_register.erb`
@ekohl Should I backport the functionality there to?